parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -98,18 +98,18 @@ public WriteBuilder forTable(Table table) {
       return this;
     }
 
-    public WriteBuilder schema(Schema schema) {
-      this.schema = schema;
+    public WriteBuilder schema(Schema setSchema) {
+      this.schema = setSchema;
```

Comment:
The convention used elsewhere to get around this problem is to use "new" as in `newSchema`. Could you use that instead for consistency?